Exercise4.rb

```diff
@@ -12,4 +12,6 @@ def palindrome(variable)
 puts " Please provide an input"
 line=gets.chomp
 Input=String.new
-print Input.palindrome(line)
\ No newline at end of file
+print Input.palindrome(line)
```

Comment:
we have to create a method called palindrome? which will not take any argument.
You know the meaning of `self` in the class